src/Microsoft.Identity.Client/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -42,13 +42,8 @@ namespace Microsoft.Identity.Client
     [EditorBrowsable(EditorBrowsableState.Never)]
     public abstract class WindowsFormsWebAuthenticationDialogBase : Form
     {
-        private RequestContext RequestContext { get; }
+        private RequestContext RequestContext { get; set; } = new RequestContext(Guid.Empty);
```

Comment:
this should not be initialized to Guid.Empty. update the code to call WindowsFormsWebAuthenticationDialogBase(RequestContext requestContext) instead. you could make the class and copnstructor internal